app.py

```diff
@@ -24,8 +26,11 @@
 app.register_blueprint(status)
 app.register_blueprint(builds)
 app.register_blueprint(auth)
+app.register_blueprint(build_enqueue)
 
 
 @app.context_processor
 def template_context():
-    return {'pipelines': config['zuul']['pipelines']}
+    return {'pipelines': config['zuul']['pipelines'],
+            'start_pipelines': config['build_enqueue']['pipelines'],
```

Comment:
Are we sure that is  the best place for this? ``start_pielines`` should be use only by ``/control`` so it not be necessary to inject this to all templates.